client/src/components/Button/Button.tsx

```diff
@@ -1,14 +1,11 @@
 import React from "react";
-import { ButtonStyles, AuthenticationButton } from "./ButtonStyles";
+import { ButtonStyles, LoginButtonStyles } from "./ButtonStyles";
 
-export default function Button() {
-  return <ButtonStyles>Search</ButtonStyles>;
-};
-
-type AuthProps = {
+type Props = {
   content: string;
+  name: string;
 }
 
-export function AuthButton({ content }: AuthProps) {
-  return <AuthenticationButton>{content}</AuthenticationButton>;
-};
+export default function Button({ content, name }: Props) {
+  return <ButtonStyles id={name + "-btn"}>{content}</ButtonStyles >;
```

Comment:
I am not a fan of using abbreviations like `btn`. I know it will make some things long worded but its so much easier to quickly understand things that you never wrote and search for things within the codebase. Also, it will make things consistent across the board because some people will use different abbreviations for the same thing.